app/views/tutorials/index.html.erb

```diff
@@ -1,16 +1,18 @@
 <main class="tutorials">
   <% @tutorials.each do |tutorial| %>
-  <section class="tutorial">
-    <div class="tutorial-thumbnail col md-col-2">
-      <a>
-        <img class"thumbnail" src= <%= "#{tutorial.thumbnail}" %> alt="">
-      </a>
-    </div>
+    <% if !current_user.nil? || !tutorial.classroom %>
```

Comment:
This is saying if currrent_user does not equal nil correct?

Reply:
Absolutely. And that tutorial.classroom is false